Nur in psi-dev-snapshot-2006-10-16.fixed: conf.log.
Nur in psi-dev-snapshot-2006-10-16.fixed: conf.pri.
Nur in psi-dev-snapshot-2006-10-16.fixed: Makefile.
Nur in psi-dev-snapshot-2006-10-16.fixed: psi.
Nur in psi-dev-snapshot-2006-10-16.fixed/src: config.h.
Nur in psi-dev-snapshot-2006-10-16.fixed/src: Makefile.
Nur in psi-dev-snapshot-2006-10-16.fixed/src: .moc.
Nur in psi-dev-snapshot-2006-10-16.fixed/src: .obj.
Nur in psi-dev-snapshot-2006-10-16.fixed/src: psi.
Nur in psi-dev-snapshot-2006-10-16.fixed/src: .rcc.
Nur in psi-dev-snapshot-2006-10-16.fixed/src: .ui.
Nur in psi-dev-snapshot-2006-10-16.fixed/third-party/libjingle: liblibjingle.a.
Nur in psi-dev-snapshot-2006-10-16.fixed/third-party/libjingle: Makefile.
Nur in psi-dev-snapshot-2006-10-16.fixed/third-party/libjingle: .obj.
diff -aurp psi-dev-snapshot-2006-10-16/third-party/libjingle/talk/base/base64.h psi-dev-snapshot-2006-10-16.fixed/third-party/libjingle/talk/base/base64.h
--- psi-dev-snapshot-2006-10-16/third-party/libjingle/talk/base/base64.h	2006-10-16 06:30:14.000000000 +0200
+++ psi-dev-snapshot-2006-10-16.fixed/third-party/libjingle/talk/base/base64.h	2006-10-16 20:44:28.000000000 +0200
@@ -22,8 +22,8 @@ public:
   static string decode(const string & data);
   static string encodeFromArray(const char * data, size_t len);
 private:
-  static const string Base64::Base64Table;
-  static const string::size_type Base64::DecodeTable[];
+  static const string Base64Table;
+  static const string::size_type DecodeTable[];
 };
 
 #endif
diff -aurp psi-dev-snapshot-2006-10-16/third-party/libjingle/talk/base/stringutils.h psi-dev-snapshot-2006-10-16.fixed/third-party/libjingle/talk/base/stringutils.h
--- psi-dev-snapshot-2006-10-16/third-party/libjingle/talk/base/stringutils.h	2006-10-16 06:30:14.000000000 +0200
+++ psi-dev-snapshot-2006-10-16.fixed/third-party/libjingle/talk/base/stringutils.h	2006-10-16 20:45:25.000000000 +0200
@@ -255,7 +255,7 @@ size_t asccpyn(wchar_t* buffer, size_t b
 template<>
 struct Traits<char> {
   typedef std::string string;
-  inline static const char* Traits<char>::empty_str() { return ""; }
+  inline static const char* empty_str() { return ""; }
 };
 
 ///////////////////////////////////////////////////////////////////////////////
diff -aurp psi-dev-snapshot-2006-10-16/third-party/libjingle/talk/xmpp/xmppclient.h psi-dev-snapshot-2006-10-16.fixed/third-party/libjingle/talk/xmpp/xmppclient.h
--- psi-dev-snapshot-2006-10-16/third-party/libjingle/talk/xmpp/xmppclient.h	2006-10-16 06:30:14.000000000 +0200
+++ psi-dev-snapshot-2006-10-16.fixed/third-party/libjingle/talk/xmpp/xmppclient.h	2006-10-16 21:12:38.000000000 +0200
@@ -133,7 +133,7 @@ private:
     }
   }
 
-  std::string XmppClient::GetStateName(int state) const {
+  std::string GetStateName(int state) const {
     switch (state) {
       case STATE_PRE_XMPP_LOGIN:      return "PRE_XMPP_LOGIN";
       case STATE_START_XMPP_LOGIN:  return "START_XMPP_LOGIN";
Nur in psi-dev-snapshot-2006-10-16.fixed/third-party/qca: libqca.a.
Nur in psi-dev-snapshot-2006-10-16.fixed/third-party/qca: Makefile.
Nur in psi-dev-snapshot-2006-10-16.fixed/third-party/qca: .moc.
Nur in psi-dev-snapshot-2006-10-16.fixed/third-party/qca: .obj.
